pages/_app.js

```diff
@@ -3,9 +3,10 @@ import GlobalStyle from "../styles";
 import Head from "next/head";
 import Movie from "../components/Movie";
 import Heading from "../components/Heading";
-import MovieDetails from "../components/Moviedetails.js";
+import MovieDetails from "../components/MovieDetail.js";
```

Comment:
You don't need to add file endings in your import statements: 
```suggestion
import MovieDetails from "../components/MovieDetail";
```
Please stick to **one** variant of import statements to keep some consistency for your code.

Reply:
hi I removed the file endings. and my features do work.you need to click on one movie then click on the heart icon to add to favorites Orr click again to remove from favorites.

Reply:
Your user story 6 has the following acceptance criteria:

- Each movie in the favorites list should have a heart icon .
- When the user clicks on the heart icon again , the movie should be removed from the user's favorites list.
- The favorites list should be updated immediately to reflect the removal of the movie.

These criteria are not yet fulfilled, that's why I mentioned to (fully) cover the functionality of your user stories in my former comment. Please adjust.

Reply:
please try again or come to our room for me to show you because it works fine for me